ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,232 @@
+"""
+The RQ integration will trace your jobs.
+
+Usage
+~~~~~
+
+The rq integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(rq=True)
+
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_SERVICE=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq
+for a working example.
+
+
+Instance Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+
+To override the service name for a queue::
+
+    from ddtrace import Pin
+
+    connection = redis.Redis()
+    queue = rq.Queue(connection=connection)
+    Pin.override(queue, service="custom_queue_service")
+
+
+To override the service name for a particular worker::
+
+    worker = rq.SimpleWorker([queue], connection=queue.connection)
+    Pin.override(worker, service="custom_worker_service")
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.rq['distributed_tracing_enabled']
+
+   If ``True`` the integration will connect the traces sent between the enqueuer
+   and the RQ worker.
+
+   Default: ``True``
+
+.. py:data:: ddtrace.config.rq['service']
+
+   The service name reported by default for RQ spans from the app.
+
+   This option can also be set with the ``DD_RQ_SERVICE`` or ``DD_SERVICE``
+   environment variables.
+
+   Default: ``rq``
+
+.. py:data:: ddtrace.config.rq_worker['service']
+
+   The service name reported by default for RQ spans from workers.
+
+   This option can also be set with the ``DD_RQ_WORKER_SERVICE`` environment
+   variable.
```

Comment:
```suggestion
   This option can also be set with the ``DD_RQ_WORKER_SERVICE`` or ``DD_SERVICE``
   environment variables.
```

?

Reply:
yes good catch. We should list DD_SERVICE first too

Reply:
`DD_SERVICE` first would be a nice touch.